openslides/motions/static/templates/motions/slide_motion.html

```diff
@@ -1,5 +1,11 @@
+<style type="text/css" ng-if="!config('motions_disable_sidebox_on_projector')">
```

Comment:
In fact that the id `motioncontent` is only used once, please use a `ng-style`.

Reply:
Done. Please review again.